### PR TITLE
Vibro Weapons

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Sprites taken from here https://github.com/d3athrow/vgstation13/pull/136

It completely bypasses armour, and (assuming my other PR is merged) will take off limbs in one hit most of the time. It can also block gunfire while wielded in two hands. Basically a very strong dual saber.

The chaplain gets his cosmetic version, which only has high AP (same properties as his scythe). The chaplains AP weapon will probably have to be weakened if the other PR goes through though (by which I mean definitely)

### CODE REVIEW

**Comment by xxalpha** [comment]
>neither PR has sprites

:bangbang:

**Comment by Core0verload** [comment]
Is there any way to punish those who make sprite PRs without sprites?

Is taking a screen shot and clickdragging it really that hard?

**Comment by KorPhaeron** [comment]
It's just a black katana, I really didn't think it was that important. The only reason sprites were mentioned at all is because credit is required.

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 1]
```diff
@@ -145,7 +145,7 @@
 	icon_state = "sord"
 	item_state = "sord"
 	slot_flags = SLOT_BELT
-	force = 2
+	force = 4.13
```
Comment (phil235) [inline code/game/objects/items/weapons/holy_weapons.dm]: wat

Comment (KorPhaeron) [inline code/game/objects/items/weapons/holy_weapons.dm]: Don't worry about it

Comment (Jordie0608) [inline code/game/objects/items/weapons/holy_weapons.dm]: I am worried

File: code/game/objects/items/weapons/twohanded.dm [hunk 2]
```diff
@@ -418,3 +416,35 @@
...
+	hitsound = 'sound/weapons/bladeslice.ogg'
+
+/obj/item/weapon/twohanded/vibro_weapon/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance, damage, attack_type)
+	if(wielded)
+		final_block_chance *= 2
+	if(wielded || attack_type != PROJECTILE_ATTACK)
+		if(prob(final_block_chance))
+			if(attack_type == PROJECTILE_ATTACK)
+				owner.visible_message("<span class='danger'>[owner] deflects [attack_text] with [src]!</span>")
+				playsound(src, pick("sound/weapons/bulletflyby.ogg","sound/weapons/bulletflyby2.ogg","sound/weapons/bulletflyby3.ogg"), 75, 1)
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/twohanded.dm]: Forgot to return 1

Comment (xxalpha) [inline code/game/objects/items/weapons/twohanded.dm]: Forgot to return 4.13

**Comment by Steelpoint** [comment]
What's the point on piercing armour when most actual armour offers little protection in the first place?

Yay, I can hack this guy to death with one half less hits than normal!

File: code/game/objects/items/weapons/holy_weapons.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/game/objects/items/weapons/holy_weapons.dm]: I worry as well

**Comment by PKPenguin321** [comment]
Whine elsewhere

File: code/game/objects/items/weapons/holy_weapons.dm (hunk 1 above)
Comment (RemieRichards) [inline code/game/objects/items/weapons/holy_weapons.dm]: I mean, it's PI to 3 significant figures backwards... does... does that mean anything here?

**Comment by KorPhaeron** [comment]
Force 40 vs a person is 3 hits to crit

Force 40 vs riot or HoS armour is ~6 hits to crit

Force 40 vs riot or HoS armour+a riot shield is ~12 hits to crit

With AP its always 3 hits, and more importantly dismembers more easily.

**Comment by Iamgoofball** [comment]
>vibro

File: code/game/objects/items/weapons/holy_weapons.dm (hunk 1 above)
Comment (KazeEspada) [inline code/game/objects/items/weapons/holy_weapons.dm]: 4/13/2009 is the start date of homestuck. It's a minor reference tacked onto a homestuck bigger reference.

Comment (Iamgoofball) [inline code/game/objects/items/weapons/holy_weapons.dm]: # NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

# NOPE

**Comment by KorPhaeron** [comment]
Putting this in the other PR

File: code/game/objects/items/weapons/holy_weapons.dm (hunk 1 above)
Comment (Jalleo) [inline code/game/objects/items/weapons/holy_weapons.dm]: Um no?

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 3]
```diff
@@ -161,6 +161,17 @@
 	sharpness = IS_SHARP
 	attack_verb = list("chopped", "sliced", "cut", "reaped")
 
+/obj/item/weapon/nullrod/scythe/vibro
+	icon_state = "hfrequency0"
+	item_state = "hfrequency1"
+	name = "high frequency blade"
+	desc = "Bad references are the DNA of the soul."
```
Comment (Jalleo) [inline code/game/objects/items/weapons/holy_weapons.dm]: Bad desc

**Pull request closed without merging**